testapi.pm

```diff
@@ -1413,6 +1413,22 @@ sub freeze_vm {
     query_isotovideo('backend_freeze_vm');
 }
 
+=head2 cont_vm
```

Comment:
what is the opposite of freeze? continue is at least part of http://www.wordhippo.com/what-is/the-opposite-of/freeze.html

But I like melt better :)

But resume_vm has its appeal too